pkg/httputil/logging.go

```diff
@@ -63,6 +66,7 @@ func DebugLoggingMiddleware(requestIDHeaderName string, fields logging.Fields) f
 				logging.MethodFieldKey:    r.Method,
 				logging.HostFieldKey:      r.Host,
 				logging.RequestIDFieldKey: reqID,
+				logging.LoggerName:        "MiddlewareLog",
```

Comment:
1. Usually we have property called "service".
2. How is this property is used? Because we didn't really create another logger and having a name can be confusing.

Reply:
Maybe "audit": "api"